ospo-mindmap/Content/ospomindmap.md

```diff
@@ -70,21 +84,80 @@ development team of the Linux Project
 
 - Security Standards
 - License Compliance Policies
-- Tooling
-  - Build in-house
-  - Adopt existing tools
-      - Commercial tools
-      - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
-- Design OSS infrastructure
-
-### OS Community Education
 
-#### Create Internal OS Ambassadors
 
-- Drive event sponsorship
-- Train speakers for coding forums
-  
-#### Create educational programs
+### 📝 Establish and Improve Open Source Policies and Processess
+
+##### Policies
+
+- Using and compliance policy
+  - Forking an open source project
+  - Copying open source licensed files/snippets into organization's codebase
+  - Mixing code (owned and third-party OSS/proprietary) and distributing as products or services (sdk, REST API, SaaS, etc.)
+  - Using standalone open source tools (e.g. Firefox, MySQL, CentOS)
+- Policy on open source licenses
+- Policy on releasing code as open source
+  - Contributing back to third party OSS project
+  - Publishing a new open source project
+  - Contributing in personal time
+  - Signing contribution agreements (e.g. CLA or DCOs)
+- Policy on making open source business decisions
+  - Risk assessment and acceptance  
+  - Business alignment
+
+#### Processes
+
+- Creating Open Source
+  - Contributing to third-party OSS projects during work time
+    - Which copyright statement to use for contributions on behalf of the organization
+    - What to do if there is already a copyright notice in the code
+    - What to do if there is no copyright notice in the code
+    - What to do if the community object to any changes to the copyright notices
+  - Contributing to third-party OSS projects during personal time
+  - Publishing a new open source project
+    - Check if the new project name is already in use in the community or is trademarked
+  - Sponsoring Open Source
```

Comment:
Add node: "Sponsoring Open Source Foundations"
And under could go:

- Linux Foundation
- Eclipse Foundation
- Open Infrastructure Foundation

Reply:
These Foundations, except Open Infrastructure Foundation, are already in the "Collaborate with Open Source Organizations" --> "Foundations" branch, but that is about collaboration, not sponsoring. Some mindmap tools allow adding links (kind of dotted line relation) from a node to another node in another branch, but that can add unnecessary complexity. Not sure if this tool has that kind of feature, but even if there is such thing, it likely just would make the already large mindmap too complex.

Reply:
@winterrocks in order to avoid extra complexity on this first iteration, what do you think about making the first section wider? 🤔  For instance, something like "Supporting Open Source Projects" could cover "sponsoring" and "collaborating" with umbrella foundations, independent foundations, and independent projects. Just an idea 🙂  (cc @tsteenbe)

Reply:
That is a good idea @anajsana. Nothing is set in stone here, so once we get this iteration ready, we can start to think the next iteration and what kind of taxonomy would work best. Again, there is no one correct answer here.

Reply:
Updated the section to the following:

```
  - Sponsoring Open Source
    - Sponsoring third-party projects or individuals
    - Sponsoring open source organizations
    - Sponsoring own organization's open source projects or individual members
    - Sponsoring events